rules-tests/DowngradePhp72/Rector/FuncCall/DowngradeJsonDecodeNullAssociativeArgRector/Fixture/possibly_null.php.inc

```diff
@@ -24,10 +24,7 @@ class PossiblyNull
 {
     function run(string $json, ?bool $associative)
     {
-        if ($associative === null) {
-            $associative = true;
-        }
-        $value = json_decode($json, $associative);
+        $value = json_decode($json, $associative === null);
     }
```

Comment:
I've changed this incorrectly, either `true` or `null` must both result in `true` - https://3v4l.org/b1mA6

I'll fix it